### ISSUE TITLE
can you provide whl every release?

### ISSUE THREAD

**Issue by iperov** [inducer/pyopencl#384]
I don't like that `pip install pyopencl` requires c++ compiler to build.

**Comment by inducer** [comment]
Technically, there is a Travis job that should do that. It doesn't currently seem to, which is weird. It gives [this build log](https://travis-ci.org/github/inducer/pyopencl/jobs/728155659), which claims that `$TWINE_USERNAME` is being set:
```
Setting environment variables from repository settings
$ export TWINE_USERNAME=[secure]
$ export TWINE_PASSWORD=[secure]
```
(Line 176)
*and* that it is not set
```
TWINE_USERNAME not set. Skipping uploading wheels
```
(Line 2583)
which seems contradictory.

@isuruf, any clues what's going on here?

As far as I can tell, the env vars are properly being passed to the Docker container.

**Comment by isuruf** [comment]
That's not a tag build. Only git tag builds are uploading wheels. Latest release at https://pypi.org/project/pyopencl/#files has the wheels

**Comment by inducer** [comment]
> That's not a tag build.

But who checks whether this is a tag build? The script inside the container? And why does it say that `TWINE_USERNAME` is not set, when the actual reason for not uploading is that it's not a tag build? (Sorry about all the questions.)

**Comment by isuruf** [comment]
Currently there's no way to set the env variables like TWINE_USERNAME just for tags, so we unset the TWINE_USERNAME env variable for non-tag builds at https://github.com/inducer/pyopencl/blob/master/.travis.yml#L21 to make sure that the wheels built on non-tag builds are not uploaded.

**Comment by inducer** [comment]
Argh. I had overlooked that. Thanks!

**Comment by inducer** [comment]
@iperov It looks like (Linux) wheels are available for 2020.2.2. Does that resolve your concern? If you would like wheels for other platforms, I'd be happy to consider a PR for CI automation to build them.
